### PR TITLE
docs: pageSizeOptions default value

### PR DESCRIPTION
**Pull request by hengkx**
https://github.com/ant-design/ant-design/blob/master/components/pagination/index.en-US.md

### CODE REVIEW

File: src/Options.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import KEYCODE from './KeyCode';
 
 class Options extends React.Component {
   static defaultProps = {
-    pageSizeOptions: ['10', '20', '50', '100'],
+    pageSizeOptions: ['10', '25', '50', '100'],
```
Comment (afc163) [inline src/Options.jsx]: 20

Comment (afc163) [reply src/Options.jsx]: https://github.com/react-component/pagination/blob/master/HISTORY.md#220

Comment (hengkx) [reply src/Options.jsx]: antd 那边 是25
image

Comment (hengkx) [reply src/Options.jsx]: image

Comment (hengkx) [reply src/Options.jsx]: 这个
imagePR changelog 没有改 要重新调整antd那边的文档了

Comment (afc163) [reply src/Options.jsx]: https://github.com/ant-design/ant-design/commit/ff95260b0a85dafc9d7dea93d31f332a8175d80c#diff-6965b769f0c97288df69652145f79c75L30

我调整了。

**Pull request merged**